integrationTests/testGameHelperFunctions.go

```diff
@@ -54,7 +54,7 @@ func PlayerJoinsGame(
 			value:    joinGameVal,
 			rcvAddr:  scAddress,
 			sndAddr:  player.Address.Bytes(),
-			data:     fmt.Sprintf("joinGame@%s", round),
+			data:     fmt.Sprintf("joinGame@00%s", hex.EncodeToString(big.NewInt(0).SetInt64(int64(round)).Bytes())),
```

Comment:
`big.NewInt(int64(round))` ?

Reply:
big.NewInt(round) does not make result correct. For small number it results in only one byte, and hex decoder in argument parser returns error.